client/app/visualizations/chart/Editor/ColumnMappingSelect.jsx

```diff
@@ -0,0 +1,60 @@
+import { isString, map, uniq, flatten, filter, sortBy, keys } from 'lodash';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'antd/lib/select';
+
+const MappingTypes = {
+  x: { label: 'X Column' },
+  y: { label: 'Y Columns', multiple: true },
+  series: { label: 'Group by' },
+  yError: { label: 'Errors column' },
```

Comment:
Looking at `queries/36#61` -> general -> Errors column
* Master allows selecting "name" and "error"
* Branch allow selecting "error" only

Is this a bug or a fix?

Reply:
In this case it's a bug fix - each column could be mapped only once; "name" is already mapped as `x`, so shouldn't be available for `error`